.github/workflows/artifact.yml

```diff
@@ -97,5 +103,5 @@ jobs:
           name: ${{ github.ref_name }}
           files: |
             release-${{ github.ref_name }}/**/*.aar
-            release-${{ github.ref_name }}/**/*.xcframework.tar.gz
+            release-${{ github.ref_name }}/**/*.xcframework
```

Comment:
Thanks for adding the iOS counterpart, @erikziyunchi! 

By inspecting the artifact built in https://github.com/gaukas/watermob/actions/runs/9868121953 I realized the uploaded artifact `Watermob.xcframework` is actually a folder containing `ios-arm64/Watermob.framework` and `ios-arm64_x86_64-simulator/Watermob.framework`. I wonder if it is possible/preferred to upload these two separately? 

Also, given that these are actually folders not single files. I am concerned what will happen if we try to directly release it. I will merge and see what happens, but at least if possible I would prefer seeing separate artifacts for ios and its simulator.

Reply:
I just checked and from what I read about `softprops/action-gh-release`, it only allows us release files, not directories.

Reply:
I think when running gomobile bind for iOS, it will generate both of them together and put them into a single folder. And `xxx.framework` is actually another folder, so we might need to make a tarball of the binary if we can't have directories for release.